_includes/popular_posts.html

```diff
@@ -0,0 +1,14 @@
+<div class="blog-popular">
+  <div class="wrapper">
+    <h2 class="header-2">Most popular on the blog</h2>
+    <div class="posts">
+      {% assign popular_posts = site.posts | where: 'popular', true %}
+
+      {% for post in popular_posts %}
+        {% assign name = "" %}
+        {% assign post = post %}
```

Comment:
why do we need this?

Reply:
Actually no we still need it, I got an error

Reply:
Actually we still need it, I got an error

Reply:
It related to view-transition, we pass this property to assing the 'view-transition-name' of each element, and if there is empty value we turn of view transiton for post what popular block has, bacause there can bee case with two posts on the whole page: on posts block and popular block and I got an error "Unexpected duplicate view-transition-name:" that is why we still need it, If I will find another way to connect transition view to popular block as well for working like for main posts block, I will add, but for now we have like default transion styles for this block. Hope it's clear

Reply:
Description from https://developer.mozilla.org/en-US/docs/Web/CSS/view-transition-name: "A distinct identifying name that causes the selected element to participate in a separate  from the root view transition. The identifier must be unique. If two rendered elements have the same view-transition-name at the same time, ViewTransition will reject and the transition will be skipped."

Reply:
ok thanks for your comment :)